src/deprecates.jl

```diff
@@ -20,7 +20,7 @@ import Base.varm, Base.stdm
 @deprecate AICc(obj::StatisticalModel) aicc(obj)
 @deprecate BIC(obj::StatisticalModel) bic(obj)
 
-@deprecate stderr(obj::StatisticalModel) se(obj)
+@deprecate stderr(obj::StatisticalModel) stderror(obj)
```

Comment:
Maybe something like
```jl
if !isdefined(Base, :stderr)
    @deprecate stderr(obj::StatisticalModel) stderror(obj)
else
    @deprecate (io::Base.TTY)(obj::StatisticalModel) stderror(obj)
end
```
so that it still mostly works in 0.7.   (Technically, the second deprecation message should be `io == stderr ? stderror(obj) : error(...)`, but that would be a rather confusing deprecation message, and in practice I don't think people are likely to pass `StatisticalModel` objects to `stdin` or `stdout`.)

Reply:
Thank you for reviewing.
The second deprecate message cause syntax error.
```
ERROR: LoadError: LoadError: invalid usage of @deprecate
```
I'm tring to fix it, but it is difficult for me.
 Could you tell me how to write?

Reply:
You should be able to work around this limitation by doing something like:
```julia
function (io::IO)(obj::StatisticalModel)
    Base.depwarn("stderr(obj::StatisticalModel) is deprecated, use stderror(obj) instead", :stderr)
    io === stderr ? stderror(obj) : throw(MethodErrror(io, (obj,)))
end
```